hbase/Dockerfile

```diff
@@ -325,7 +325,7 @@ rm -rf /var/cache/yum
 
 ln -s /stackable/hbase-${PRODUCT} /stackable/hbase
 ln -s /stackable/hbase-operator-tools-${HBASE_OPERATOR_TOOLS} /stackable/hbase-operator-tools
-ln -s "/stackable/phoenix/phoenix-server-hbase-${HBASE_PROFILE}-${PHOENIX}.jar" "/stackable/hbase/lib/phoenix-server-hbase-${HBASE_PROFILE}-${PHOENIX}.jar"
```

Comment:
I think ideally the `ln` command should fail when the target does not exist - this would have catched the error
I tested it locally and `--logical` seems like the thing we want, but I'm no `ln` expert
```
ln --verbose --logical foo bar
ln: failed to access 'foo': No such file or directory
```

Reply:
Done https://github.com/stackabletech/docker-images/pull/811/commits/5ecdb565e7e9cac4293fa9193e346d374f4696ec